zerver/lib/exceptions.py

```diff
@@ -271,6 +272,17 @@ def msg_format() -> str:
         return _("Must be an organization or stream administrator")
 
 
+class UserDeactivatedError(JsonableError):
+    code: ErrorCode = ErrorCode.USER_DEACTIVATED
+
+    def __init__(self) -> None:
+        pass
```

Comment:
Do we need this `__init__` method?  I haven't read this framework recently, and it seems a bit odd that most of our custom errors need to override it.

Reply:
Yeah I think that's required because if we do not explicitly define it, then when we do `UserDeactivatedError()` python will find no `__init__` method so it will go up in mro hierarchy to call `__init__` method in `JsonableError` call it without `msg` parameter and that will give
```python
TypeError: __init__() missing 1 required positional argument: 'msg'
```
so we need to pass `__init__` in child classes.

Reply:
Ahh, OK, makes sense.